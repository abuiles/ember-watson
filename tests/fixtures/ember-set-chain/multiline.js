function fn() {
  x.set('foo', 0)
   .set('bar', 1)
   .set('baz', 2);

  x.set('foo', {
    a: 0
  }).set('bar', 1);
}
