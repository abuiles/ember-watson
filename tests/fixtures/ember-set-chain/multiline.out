Using the return value of `set`:

  1 | function fn() {
  2 |   x.set('foo', 0)
  3 |    .set('bar', 1)
    |   ^^^^^^^^^^^^^^^
  4 |    .set('baz', 2);

Using the return value of `set`:

  1 | function fn() {
  2 |   x.set('foo', 0)
    |   ^^^^^^^^^^^^^^^
  3 |    .set('bar', 1)

Using the return value of `set`:

  5 | 
  6 |   x.set('foo', {
  7 |     a: 0
  8 |   }).set('bar', 1);
    |   ^^
  9 | }
